src/app/Caches/Create/ConfigurationFeature.tsx

```diff
@@ -0,0 +1,275 @@
+            evictionType: evictionType,
+            maxSize: maxSize,
+            maxCount: maxCount,
+            evictionStrategy: evictionStrategy,
+        });
+
+        if (evictionType === EvictionType.size)
+            props.handleIsFormValid(parseInt(maxSizeNumber) >= 0)
+        else if (evictionType === EvictionType.count)
+            props.handleIsFormValid(parseInt(maxCount) >= 0)
+        else if (cacheFeatureSelected.length < 1)
+            props.handleIsFormValid(true);
+        else
+            props.handleIsFormValid(false);
+
+    }, [cacheFeatureSelected, evictionType, maxSize, maxCount, evictionStrategy]);
+
+    const onSelect = (event, selection) => {
+        if (cacheFeatureSelected.includes(selection)) {
+            setCacheFeatureSelected(cacheFeatureSelected.filter(item => item !== selection));
+        } else {
+            setCacheFeatureSelected([...cacheFeatureSelected, selection]);
+        }
+    };
+
+    const clearSelection = () => {
+        setCacheFeatureSelected([]);
+        setIsOpenCacheFeature(false);
+    };
+
+    const onSelectEvictionStrategy = (event, selection, isPlaceholder) => {
+        setEvictionStrategy(selection);
+        setIsOpenEvictionStrategy(false);
+    };
+
+    const onSelectMaxCountUnit = (event, selection, isPlaceholder) => {
+        setMaxSizeUnit(selection);
+        setIsOpenMaxSizeUnit(false);
+    };
+
+    const cacheFeatureOptions = () => {
+        return Object.keys(CacheFeature).map((key) => (
+            <SelectOption key={key} value={CacheFeature[key]} isDisabled={key !== 'BOUNDED'} />
+        ));
+    };
+
+    const formCacheFeatureSelect = () => {
+        return (
+            <FormGroup fieldId='cache-feature'>
+                <MoreInfoTooltip label={t('caches.create.configurations.feature.cache-feature-list', { brandname: brandname })} toolTip={t('caches.create.configurations.feature.cache-feature-list-tooltip')} textComponent={TextVariants.h2} />
+                <Select
+                    variant={SelectVariant.typeaheadMulti}
+                    typeAheadAriaLabel={t('caches.create.configurations.feature.cache-feature-list-typeahead')}
+                    onToggle={() => setIsOpenCacheFeature(!isOpenCacheFeature)}
+                    onSelect={onSelect}
+                    onClear={clearSelection}
+                    selections={cacheFeatureSelected}
+                    isOpen={isOpenCacheFeature}
+                    aria-labelledby="cache-feature"
+                    placeholderText={t('caches.create.configurations.feature.cache-feature-list-placeholder')}
+                >
+                    {cacheFeatureOptions()}
+                </Select>
+            </FormGroup>
+        )
+    }
+
+    // Options for Eviction Strategy
+    const evictionStrategyOptions = () => {
+        return Object.keys(EvictionStrategy).map((key) => (
+            <SelectOption key={key} value={EvictionStrategy[key]} />
+        ));
```

Comment:
@dipguptaredhat Hey, one comment about the eviction strategy drop-down. I think it should show only REMOVE and EXCEPTION. It doesn't make sense to set NONE as the eviction strategy because that means eviction won't happen so is kind of a misconfiguration. And MANUAL is for embedded caches only. From the server there is no way for users to manually invoke `evict()` on a remote cache.